.github/actions/run-fixtures-test/action.yml

```diff
@@ -0,0 +1,52 @@
+name: Run a fixtures test
+description: Run a fixtures test
+
+inputs:
+  fixtures-dir:
+    description: Path to the fixtures directory
+    required: true
+  command:
+    description: The git-cliff command to run
+    required: false
+    default: ""
+
+runs:
+  using: composite
+  steps:
+    - name: Install toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: nightly
+        override: true
+
+    - name: Install git-cliff
+      run: |
+        cargo install --path git-cliff/
+      shell: bash
+
+    - name: Set git config
+      run: |
+        git config --global user.name github-actions
+        git config --global user.email github-actions@github.com
```

Comment:
You can use the following config to commit as the original github-actions bot:

```
git config user.name 'github-actions[bot]'
git config user.email 'github-actions[bot]@users.noreply.github.com'
```

This was something that I learned today so I thought I'd share. No change required here :D

Reply:
Thank you for sharing it!
Followed the style in c5a39ca964a31531c5730f05254f0af7fbc86d5c.

Reply:
`--global` was necessary. Also, will use `""` as shown in the error message.

https://github.com/orhun/git-cliff/runs/5088222621?check_suite_focus=true#step:3:350

```
Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"
```